enrichment/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3
+
+COPY ./requirements.txt /app/requirements.txt
+WORKDIR /app
+RUN pip install -r requirements.txt
+COPY . /app/enrichment
+
+ENV PYTHONPATH "${PYTHONPATH}:/app"
+CMD [ "python", "./enrichment/app.py" ]
```

Comment:
Won't this cause to run the enrichment via the python interpreter which runs on a single thread and eventually we will handle a single file each time, no?

Look at [here](https://github.com/hasadna/open_pension/blob/master/processor/Dockerfile) on how we set up the flask app. It require a bit of file changes.